js/runner.go

```diff
@@ -440,5 +440,14 @@ func (u *VU) runFn(ctx context.Context, group *lib.Group, fn goja.Callable, args
 
 	state.Samples <- u.Dialer.GetTrail(startTime, endTime, isFullIteration, stats.IntoSampleTags(&tags))
 
+	// If MinIterationDuration is specified and the iteration wasn't cancelled
+	// and was less than it, sleep for the remainder
+	if state.Options.MinIterationDuration.Valid {
```

Comment:
Shouldn't we check isFullIteration here ? In order to not count cancelled iterations